includes/class-wc-stripe-order-handler.php

```diff
@@ -43,77 +43,27 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Disables the ability to edit for orders with uncaptured payment.
+	 * Shows a warning message about editing uncaptured orders.
 	 *
-	 * @param $editable boolean The current editability of the order.
-	 * @param $order WC_Order The order object.
-	 * @return boolean false if the order has uncaptured payment, true otherwise.
+	 * @param $order_id
 	 */
-	public function disable_edit_for_uncaptured_orders( $editable, $order ) {
+	public function show_warning_for_uncaptured_orders( $order_id ) {
+		$order = wc_get_order( $order_id );
 		// Bail if payment method is not manual capture supporting stripe method.
 		if ( ! WC_Stripe_Helper::payment_method_allows_manual_capture( $order->get_payment_method() ) ) {
 			return $editable;
 		}
 
 		try {
 			$intent = $this->get_intent_from_order( $order );
-
 			if ( $intent && 'requires_capture' === $intent->status ) {
```

Comment:
We can make use of the new constant class here `WC_Stripe_Intent_Status::REQUIRES_CAPTURE`